### PR TITLE
Avatar+Badge component done

### PR DESCRIPTION
**Pull request by yogesh-**

### CODE REVIEW

File: avatar_badge.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    width: 2rem;
+    border-radius: 50%;
+    background-color: green;
+    position: absolute;
+    right: 0;
+    bottom: 8%;
+    border: 3px solid white;
+  }
+  
+  .offline{
```
Comment (akash-1318) [inline avatar_badge.css:42]: for  .online and .offline class you are repeating the same code for changing the background color just add a new class

File: avatar_badge.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    width: 2rem;
+    border-radius: 50%;
+    background-color: red;
+    position: absolute;
+    right: 0;
+    bottom: 8%;
+    border: 3px solid white;
+  }
+  
+  .dnd{
```
Comment (syed-shaida-hussain) [inline avatar_badge.css:53]: use a proper class name for dnd.

Comment (akash-1318) [inline avatar_badge.css]: same for this also

File: avatar_badge.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    height: 2rem;
+    width: 2rem;
+    border-radius: 50%;
+    background-color: gray;
+    position: absolute;
+    right: 0;
+    bottom: 8%;
+    border: 3px solid white;
+  }
```
Comment (syed-shaida-hussain) [inline avatar_badge.css:54]: use a proper class name for .dnd.

File: avatar_badge.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    top: 8%;
+    color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-weight: bold;
+    
+  }
+  
+  .number-blue{
```
Comment (akash-1318) [inline avatar_badge.css:73]: for  .number-red and  .number-blue for these classes you are also repeating the code just avoid repeatation of code and else is fine

File: avatar_badge.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    background-color: red;
+    position: absolute;
+    right: -5%;
+    top: 8%;
+    color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-weight: bold;
+
```
Comment (syed-shaida-hussain) [inline avatar_badge.css:70]: use prettier in your code for proper indentation.

File: avatar_badge.css (hunk 4 above)
Comment (yogesh-) [reply avatar_badge.css:73]: Done the changes.

File: avatar_badge.css (hunk 1 above)
Comment (yogesh-) [reply avatar_badge.css:42]: done

File: avatar_badge.css (hunk 2 above)
Comment (yogesh-) [reply avatar_badge.css]: done

File: avatar_badge.css (hunk 5 above)
Comment (yogesh-) [reply avatar_badge.css:70]: done

**Pull request merged**